scripts/adify

```diff
@@ -28,7 +28,7 @@ function adRequirePath(srcFilename) {
   var dir = path.dirname(srcFilename);
   var adFileName = path.join(__dirname, '..', 'src', 'ad.js');
   var adPath = path.normalize(path.dirname(path.relative(dir, adFileName)));
-  return adPath + path.sep + 'ad.js';
+  return adPath + '/' + 'ad.js';
 }
```

Comment:
It seems odd to me that this worked. I used `path.sep` precisely because I hoped it would work cross-platform. Any idea what's happening?